app/routes/component-tree.js

```diff
@@ -0,0 +1,47 @@
+import TabRoute from "ember-inspector/routes/tab";
+
+export default TabRoute.extend({
+  model() {
+    return [];
```

Comment:
I'm curious, do we need to return an empty array here?

Reply:
I think it was copy-pasta from the `view-tree` based on my recollection, but I'll check.